app/src/main/java/com/github/se/eduverse/ui/quizz/QuizzScreen.kt

```diff
@@ -0,0 +1,326 @@
+package com.github.se.eduverse.ui.quizz
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.github.se.eduverse.api.Question
+import com.github.se.eduverse.repository.QuizzRepository
+import com.github.se.eduverse.ui.navigation.NavigationActions
+import kotlinx.coroutines.launch
+
+// Define color constants
+private val COLOR_TEXT_PLACEHOLDER = Color(0xFF0D8899).copy(alpha = 0.6f)
+private val COLOR_CORRECT = Color(0xFF008E21)
+private val COLOR_INCORRECT = Color(0XFFCC0909)
```

Comment:
It's a good behavior defining constants for magic numbers here but you can use the Color.kt file we already have for this

Reply:
Good remark, I will do it.